markdown.dtx

```diff
@@ -14915,6 +14923,12 @@ function M.writer.new(options)
     local name = util.cache(options.cacheDir, s, nil, nil, ".verbatim")
     return {"\\markdownRendererInputFencedCode{",name,"}{",i,"}"}
   end
+  function self.jekyllData(s)
+    if not self.is_writing then return "" end
+    s = string.gsub(s, '([^:]+):%s([^\r\n]+)[\r\n]+', '\\markdownRendererSetJekyllData{%1}{%2}')
```

Comment:
We will want to surround the `\markdownRendererSetJekyllData` with corresponding `Begin; and `End` commands, so that the users can react to the fact that all the metadata has been set by the user.